lib/hammer_cli_katello/acs_bulk_actions.rb

```diff
@@ -0,0 +1,38 @@
+module HammerCLIKatello
+  class AcsBulkActionsCommand < HammerCLIKatello::Command
+    desc 'Modify alternate content sources in bulk'
+    resource :alternate_content_sources_bulk_actions
+
+    class RefreshAllCommand < HammerCLIKatello::SingleResourceCommand
+      action :refresh_all_alternate_content_sources
```

Comment:
Does this really work? I mean in the main Katello PR it's `/alternate_content_sources/bulk/refresh_all`, so here it should be `resource :bulk`, `action :refresh_all`...

Reply:
It does. That's one of the things that was so tricky about the PR; hammer pulls the URL from the apidocs. If you click on the refresh all action in the APIdocs, at the top of the page it will show you the APIdoc location of the resource, `alternate_content_sources_bulk_actions/refresh_all_alternate_content_sources`. I think it pulls these from the function names in Katello.

Reply:
At least that's the conclusion I came to after a bunch of testing the last few days ¯\_(ツ)_/¯

Reply:
Since the bulk refresh and destroy actions already existed, I don't think it makes sense to change API endpoints in Katello, at least right now.